packages/core/src/content-delivery/ContentDelivery.ts

```diff
@@ -264,7 +264,7 @@ export function createClient(
                     [`locale`]: localeCode ? { $in: [localeCode] } : undefined,
                     [`status`]: { $in: [Story.Status.PUBLISHED] },
                     [`visibility`]: { $in: [Story.Visibility.PUBLIC] },
-                    [`tag_names`]: tags?.length ? { $in: [tags] } : undefined,
+                    [`tag.name`]: tags?.length ? { $any: [tags] } : undefined,
```

Comment:
What's the difference between `any` and `in` in case of search predicate?

Reply:
Would `tags?.length ? { $in: tags } : undefined` also work?

Reply:
This particular filter only supports `$any`, `$some` and `$none` filters, $in is not supported

Reply:
But is there any difference between those predicates? 🤔

Reply:
I'm not sure there is